lib/graphgists.js

```diff
@@ -278,7 +278,7 @@ function assignPageAttributes(graphGists, contentCatalog, asciidocConfig) {
         'page-slug': graphGist.slug,
         'page-industries': allIndustries,
         'page-use-cases': allUseCases,
-        'page-browserGuideUrl': `neo4j-desktop://graphapps/neo4j-browser?cmd=play&arg=https://guides.neo4j.com/graph-examples/${graphGist.slug}/graph_guide `
+        'page-browserGuideUrl': `https://browser.neo4j.io/?cmd=play&arg=https://guides.neo4j.com/graph-examples/${graphGist.slug}/graph_guide`
```

Comment:
Actually, I think we should use:

```suggestion
        'page-browserGuideUrl': `https://guides.neo4j.com/graph-examples/${graphGist.slug}/graph_guide`
```

An then, we can reconstruct the complete URL in the UI using https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams